File: cses/introductoryProblems/trailingZeros.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(void)
{
	long long n, count = 0;
	cin >> n;
	for (int i = 5; n >= i; i = i * 5)
	{
		count = count + floor(n/i);
	}
	cout << count << endl;
	return 0;
}

File: cses/introductoryProblems/trailingZeros.java
import java.util.Scanner;

public class trailingZeros
{
	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		long n = sc.nextLong();

		int count = 0;

		for (int i = 5; n >= i; i = i * 5)
		{
			count = count + (int)(Math.floor(n/i));
		}

		System.out.println(count);
		sc.close();
	}
}